scripts/release.js

```diff
@@ -111,6 +110,34 @@ fs.writeFileSync(
 spawnOrFail('npm', [`version ${versionString} --no-git-tag-version`]);
 logger.log(`Updated package.json version to ${versionString}`);
 
+
+// Udpate meeting demo to the most up to date version of the SDK
+process.chdir(path.join(__dirname, '../demo/meeting'));
+spawnOrFail('npm', ['uninstall amazon-chime-sdk-js']);
+spawnOrFail('npm', ['install amazon-chime-sdk-js']);
+
+// Update the chat demo to the most up to date version of the SDK
+process.chdir(path.join(__dirname, '../demo/chat'));
+spawnOrFail('npm', ['uninstall amazon-chime-sdk-js']);
+spawnOrFail('npm', ['install amazon-chime-sdk-js']);
+
+// Update the peer dependency to the most updated version of the SDK
+let demoPackageJson = JSON.parse(fs.readFileSync('package.json', 'utf-8'));
+const updatedSdkVersion = demoPackageJson.dependencies['amazon-chime-sdk-js'];
+
+logger.log(`Installing SDK Version: ${updatedSdkVersion} into the meeting demo, chat demo, and as a peerDependency and devDependency of the react library.`);
+
+process.chdir(path.join(__dirname, '..'));
+let componentsPackageJson = JSON.parse(fs.readFileSync('package.json', 'utf-8'));
```

Comment:
Also want to update devDependency too

Reply:
Yup, we update the devDep in this script. I added it